packages/sdk/src/swap/v2/types.ts

```diff
@@ -76,8 +76,8 @@ interface EgressFields {
   scheduledAt: number;
   scheduledBlockIndex: string;
   txRef: string | undefined;
-  confirmedAt: number | undefined;
```

Comment:
`confirmedAt` is a bit ambiguous because its also called `confirmed` when a transaction is included in a block

Reply:
i think `witnessedAt` is a nice alternative bc you find something helpful when you google for `chainflip witness` and we can also use it for deposits

Reply:
but we also have a concept of # of confirmations required